webpack-demo/src/listFunctions.js

```diff
@@ -0,0 +1,93 @@
+// eslint-disable-next-line import/no-cycle
+import { showList } from './index.js';
+
+// eslint-disable-next-line import/no-mutable-exports
```

Comment:
- [ ] Kindly make sure to refrain from using eslint disable as it defies the purpose of using linters in the first place.